launch/launch/utilities/type_utils.py

```diff
@@ -152,7 +152,7 @@ def extract_type(data_type: AllowedTypesType) -> Tuple[ScalarTypesType, bool]:
     """
     is_list = False
     scalar_type: ScalarTypesType = cast(ScalarTypesType, data_type)
-    if is_typing_list(data_type) and data_type.__args__:
+    if is_typing_list(data_type) and hasattr(data_type, '__args__'):
```

Comment:
@clalancette this is not strictly the same, 
```suggestion
    if is_typing_list(data_type) and getattr(data_type, '__args__'):
```
is.

Reply:
While true, it is that behavior that Python 3.9 is choking on.  Even switching it to `getattr` causes it to still fail with:

```
______________________________ test_extract_type _______________________________
test/launch/utilities/test_type_utils.py:72: in test_extract_type
    extract_type(List)
launch/utilities/type_utils.py:155: in extract_type
    if is_typing_list(data_type) and getattr(data_type, '__args__'):
/usr/lib/python3.9/typing.py:710: in __getattr__
    raise AttributeError(attr)
E   AttributeError: __args__
```

I guess I could do:

```suggestion
    if is_typing_list(data_type) and hasattr(data_type, '__args__') and getattr(data_type, '__args__'):
```

What do you think?

Reply:
Ah, I missed the default value `getattr(data_type, '__args__', None)`!

Your suggestion would work as well.

Reply:
Ah, I like your suggestion better.  3a826ea implements that.